src/pages/lessons/[slug].tsx

```diff
@@ -52,6 +52,29 @@ import EmailCaptureCtaOverlay from '../../components/pages/lessons/overlay/email
 
 const tracer = getTracer('lesson-page')
 
+const specialLessons = {
```

Comment:
this is just a hack to identify a specific lesson to show some stuff when it's over

not very in depth, just noticed this lesson was getting a LOT of google flow